keymanapp/keyman
# [Android] Accommodate dimension for device banner

This PR does some cleanup and integrates suggestions with the Android app

* Per @jahorton , allow the apps to specify `Banner.DEFAULT_HEIGHT`.
* Add listener for `mm.modelchange` event to detect model changes "unload" and "load"
* Have Android app track the banner (suggestion vs blank) on lexical model change
* Pass the banner height to KMW on device rotations (portrait vs landscape)
* Update build scripts so `-debug` variant uses unminified KMW

### Testing Steps ###
1. Install Joshua's en wordlist `/web/testing/prediction-ui/simple-en-wordlist.js`
2. Add another English (US) keyboard
3. On the device, delete the en.custom model that gets installed (it currently generates exceptions)
4. The suggestion banner should appear when switching to an English keyboard. Accepting suggestions will have the same functionality as the KMW test page
5. The suggestion banner should disappear when switching to a non-English keyboard
